### PR TITLE
BaSP-TG-113: Employee Project screen

### PR DESCRIPTION
**Pull request by iaracriscenti**
### Description

An Employee  should be able to see all the projects in which he participates or, if not, warn that he does not have one assigned.

If the employee is PM (Project Manager) on any project, a button will appear that allows to list all the projects in which he's a PM and edit them.

All rows have the redirection option that will open the screen of the selected project with its information.

### CODE REVIEW

**Review by Javier-Cervera — APPROVED** [review]
Looks great!
Good job with Fragments

File: src/Components/Employee/Projects/Form/index.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+      'string.empty': 'Project name is a required field'
+    }),
+  description: Joi.string()
+    .pattern(/(.*[a-zA-Z]){4}/)
+    .required()
+    .messages({
+      'string.pattern.base': 'Description must contain at least 4 letters',
+      'string.empty': 'Description is a required field'
+    }),
+  startDate: Joi.date().min('01-01-1950').max('12-31-2050').required().messages({
```
Comment (agusting19) [inline src/Components/Employee/Projects/Form/index.js:30]: All dates are in format dd-mm-yyyy and that does not allow me to view the deployment. Dates need to be in format dd/mm/yyyy

**Review by agusting19 — APPROVED** [review]
Everything looks good but i couldn't test it

File: src/Components/Employee/Projects/Form/index.js (hunk 1 above)
Comment (iaracriscenti) [reply src/Components/Employee/Projects/Form/index.js:30]: Thanks for the reminder!

**Review by ArianaMaldonado — APPROVED** [review]
**Test Result: Passed ✅**

**Pull request merged**